site/templates/message.html

```diff
@@ -43,7 +46,7 @@ <h2>Send Brian a message on his Tidbyt.</h2>
 
         <div class="mb-3">
           <label for="FONT" class="form-label">Font</label>
-          <select name="FONT" id="FONT" maxlength=100 class="form-select">
+          <select name="FONT" id="FONT" maxlength="100" class="form-select">
```

Comment:
This, and a few other spots, were auto-formatting changes. Attributes should always have quotes around them